bricks.html

```diff
@@ -0,0 +1,21 @@
+<html>
+<body>
+<script src="bricks.js"></script>
+<!-- <div style="width: 100px; height: 50px">
+	<div id="rectangle" style="width: 50px; height: 50px; background-color: #B93838; position: absolute; border-style: solid hidden solid solid; border-color: white" onclick="borderclick(this)">
+	</div>
+	<div id="rectangle" style="width: 50px; height: 50px; background-color: #B93838; position: absolute; left: 60px; border-style: solid solid solid hidden; border-color: white" onclick="borderclick(this)">
+	</div>
+	<p style="position: absolute; z-index: 2; font-size: 12px">I'm a center brick</p>
+</div>
+-->
+<svg width="120" height="70">
+	<rect x="10" y="10" width="100" height="50" style="fill:rgb(192,75,75)" />
+	<rect x="10" width="50" height="10" style="fill:white" onmouseover="bordermouse(this)" onmouseout="borderunmouse(this)" onclick="borderclick(this)" /> <!-- top left border -->
```

Comment:
remove the `this` parameter here, so that you have `onmouseover="bordermouse"`. Then you can remove the `id` parameter to your event handlers, and refer directly to `this` instead.

Reply:
Hmm, not sure if I'm interpreting this correctly? When I remove the `this` as a parameter and sub `id` for `this` in the js function it doesn't like it, javascript console gives me "Cannot read property 'fill' of undefined" when I mouseover.